03/app.js

```diff
@@ -1,14 +1,29 @@
-console.log('DOM');
+// console.log('DOM');
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+	attr: {
+		className: 'btn',
+		title: 'super button',
+	},
+	css: {
+		border: '1px solid #336699',
+		padding: '5px 20px',
+		color: '#444',
+	},
+	text: 'Click me!',
+};
+
+const sectionElem = document.querySelector('.parent-for-button');
+const btnElement = document.createElement('button');
+
+for (const attrKeys in buttonSettings.attr) {
+	btnElement.setAttribute(attrKeys, buttonSettings.attr[attrKeys]);
+}
+for (const cssKeys in buttonSettings.css) {
+	btnElement.style[cssKeys] = buttonSettings.css[cssKeys];
+}
+for (const textKeys in buttonSettings.text) {
+	btnElement.textContent = buttonSettings.text;
+}
```

Comment:
Czy faktycznie tutaj potrzebujemy pętli? Czy przypadkiem, krócej nie będzie `btnElement.textContent = buttonSettings.text;`? - to w kontekście pośpiechu, o którym pisałem ostatnio w CR :)

Reply:
w sumie na poczatku tak mialem, ale uznalem ze napisze sobie petla zeby utrwalic syntax petli for in